integration-cli/docker_cli_daemon_plugins_test.go

```diff
@@ -197,10 +197,21 @@ func (s *DockerDaemonSuite) TestVolumePlugin(c *check.C) {
 	if err != nil {
 		c.Fatalf("Could not install plugin: %v %s", err, out)
 	}
+	pluginID, err := s.d.Cmd("plugin", "inspect", "-f", "{{.Id}}", pName)
+	pluginID = strings.TrimSpace(pluginID)
+	if err != nil {
+		c.Fatalf("Could not retrieve plugin ID: %v %s", err, pluginID)
+	}
 	defer func() {
 		if out, err := s.d.Cmd("plugin", "disable", pName); err != nil {
 			c.Fatalf("Could not disable plugin: %v %s", err, out)
 		}
+		c.Log("plugin disabled")
+		mounts, err := mount.GetMounts()
+		c.Assert(err, checker.IsNil)
+		for _, mnt := range mounts {
```

Comment:
We are simply printing out the mounts, which doesnt test much. Can we use `findmnt` to see if the mount propagated?